apps/odyssey-stats/src/styles/wp-admin.scss

```diff
@@ -89,13 +89,4 @@
 	ul.wp-submenu-wrap {
 		margin-left: 0;
 	}
-
-	&.wp-core-ui .button.is-secondary {
```

Comment:
These turned out not necessary after we replaced @automattic/components with @wordpress/components

cc/ @tyxla @mtias